### DISCUSSION TITLE
Fatality AB350 ITX - 0.8.7-2nd RELEASE

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASRock_Fatality_AB350_Gaming-ITX_Hackintosh#3] [Announcements]
### News / Fixes:
- LAN fixed thanks to new AppleIGB kext by @mbarbierato
- WiFi more reliable thanks to new Airportitlwm.kext alpha release for macOS Ventura

### Issues:
- Microphone, and Line In ports are currently not working at all with AppleALC and VoodooHDA requires SIP & Gatekeeper disabled

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/ASRock_Fatality_AB350_Gaming-ITX_Hackintosh_OpenCore/releases/tag/0.8.7v2'>Fatality AB350 ITX - 0.8.7-2nd RELEASE</a>.</em>

**Comment by Knives79** [comment]
Thanks for the release, I built myself an Opencore EFI, but with the latest version (0.9.0) I have some issues with my RX5600XT, when I launch MacOS it have a black screen for 30 seconds between the apple logo and the user login prompt, if I try to launch Windows 11 the graphic card give me an Error 43 and doesn't work (only in legacy VGA mode)... Did you ever experienced something like that? Thanks.

**Reply by RobyRew** [reply]
I have RX560 on my system, and I'm not experiencing anything like that.

I have added an AGPMInjector.kext for my RX560 too maybe you should replace it or create a new one for your GPU and replace then with the one on the repo.
https://github.com/RobyRew/AGPMInjector/releases/tag/3.3.6
![image](https://user-images.githubusercontent.com/49403384/231116997-98468911-9fa7-48dd-8074-218914cae088.png)

Kext for RX5600XT:
[AGPMInjector.kext.zip](https://github.com/RobyRew/ASRock_Fatality_AB350_Gaming-ITX_Hackintosh_OpenCore/files/11198856/AGPMInjector.kext.zip)
